packages/runner-ct/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@packages/runner-ct",
+  "version": "0.0.0-development",
+  "private": true,
+  "main": "index.js",
+  "browser": "src/index.js",
+  "scripts": {
+    "build": "webpack",
+    "build-prod": "cross-env NODE_ENV=production yarn build",
+    "clean-deps": "rm -rf node_modules",
+    "postinstall": "echo '@packages/runner needs: yarn build'",
+    "test-unit": "ts-mocha --config ./test/.mocharc.js",
```

Comment:
Remove and replace `ts-mocha` with `mocha -r @packages/ts/register`

Reply:
Will we still get compilation errors in the test runs?

Reply:
Seemingly -- if not, all of the type errors in server would not have failed in the feature branch